tools/code-tools/jtreg.sh

```diff
@@ -17,6 +17,7 @@ readonly JTREG_7_1='jtreg-7.1.1+1'
 readonly JTREG_7_2='jtreg-7.2+1'
 readonly JTREG_7_3='jtreg-7.3+1'
 readonly JTREG_7_3_1='jtreg-7.3.1+1'
+readonly JTREG_7_4_1='jtreg-7.4+1'
```

Comment:
This should be `JTREG_7_4` to match the `7.3+1` variable.

Reply:
Fixed in e4b2975bad7b43a5798e0702a355b8a74fcf65dc.